backend/app/src/main/kotlin/io/tolgee/api/v2/controllers/V2UserController.kt

```diff
@@ -35,6 +58,13 @@ class V2UserController(
     return privateUserAccountModelAssembler.toModel(userAccount)
   }
 
+  @Operation(summary = "Deletes current user.")
+  @DeleteMapping("")
+  @NeedsSuperJwtToken
```

Comment:
I actually added the super JWT stuff because of this. Once you have super token, you can do anything. It already does minimalize the probability someone do this on my computer to acceptably low probability.

Reply:
While the SJWT does handle the security part, I'm thinking an actual re-authentication (password + mfa if enabled) does increase the level of protection on the endpoint, and doubles as a way to force the user to re-authenticate in the UI as a mean of making sure they are aware of what they're doing and the consequences.

Reply:
I know. I just don't really want to fix it now. So let's create an issue and leave it for future me.